### PR TITLE
Tabs Update --> tabs reorganized with new structure, some text changes, and additional criteria

### PR DESCRIPTION
**Pull request by elbfox**
Shifts structure from alternatives by criteria to dams by criteria. Tabs = dams, to be collapsed later on.

### CODE REVIEW

**Comment by elbfox** [comment]
@sythel please review and let me know if it is safe to merge. There are no conflicts by my tabs adjustment is considerable, so I would prefer to have a second set of eyes on it before merging. Also, were you able to archive a link for the existing dev site : http://shiny.gsscdev.com/dams_mcda/ ? I don't want to merge these changes before we can have some kind of archived, working link for Sharon.

**Comment by elbfox** [comment]
moved the raw preference outputs to individual dam results tabs. 

TODO: edit graphs

File: src/dams_mcda/server.R [hunk 1]
```diff
@@ -366,48 +358,40 @@ server <- function(input, output, session) {
...
+			input$Justice4
 			)
 
-		# create table matrix 1x5
-		Alt4_Table <- as.matrix(data.frame(Alt4))
-		row.names(Alt4_Table) <- criteria_names
-		names(Alt4_Table) <- "Raw Score"
-
-		# results
-		output$SummPlot4 <- renderBarPlot(
```
Comment (sythel) [inline src/dams_mcda/server.R]: We will want the graph generations here, but I have simplified them in my branch so deleting them here is fine

Comment (elbfox) [reply src/dams_mcda/server.R]: ok I think that this might be resolved in the WSMUpdate_Dams --> TabsUpdate_Dams merge.

**Pull request merged**